CHANGELOG.md

```diff
@@ -11,4 +11,5 @@
   * remove root key option and split JSON adapter [@joaomdmoura]
   * adds FlattenJSON as default adapter [@joaomdmoura]
   * adds support for `pagination links` at top level of JsonApi adapter [@bacarini]
-  * adds extended format for `include` option to JSONAPI adapter [@beauby]
+  * adds extended format for `include` option to JsonApi adapter [@beauby]
+  * adds support for top level jsonapi member support [@beauby]
```

Comment:
There's an extraneous "support" at the end. Plus it should read `[@beauby, @bf4]` :)